core/drivers/sub.mk

```diff
@@ -39,5 +39,6 @@ srcs-$(CFG_IMX_RNGB) += imx_rngb.c
 
 subdirs-y += crypto
 subdirs-$(CFG_BNXT_FW) += bnxt
+subdirs-$(CFG_GENERIC_CLK) += clk
```

Comment:
To me "generic" goes without saying. How about `CFG_DRIVERS_CLK` or `CFG_CORE_CLK` instead?

Reply:
CFG_DRIVERS_CLK_SUPPORT might be better seems it implies that there is some support for clocks ?

Reply:
OK, sounds good.